modules/appeals_api/app/swagger/appeals_api/v1/schemas/notice_of_disagreements.rb

```diff
@@ -27,6 +27,7 @@ def self.nod_json_schemas
         }
       end
 
```

Comment:
I added these these rubocop disable/enable changes in NOD v1 files because they keep showing up for me as errors when I run `bundle exec rubocop -c ./.rubocop.yml modules/appeals_api/` locally - they don't cause a problem for PRs, but they've been making parsing rubocop outputs locally a bit annoying for a while